src/EditorFeatures/CSharpTest/GoToNextAndPreviousMember/CSharpGoToNextAndPreviousMemberTests.cs

```diff
@@ -0,0 +1,517 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests
+{
+    public class CSharpGoToNextAndPreviousMemberTests
+    {
+        [Fact, Trait(Traits.Feature, Traits.Features.GoToNextAndPreviousMember)]
```

Comment:
Are we okay with no [WorkItem] attributes on initial feature work, and then only using them for bugfixes?